eventfiles/class_tests/event_file_views_tests.py

```diff
@@ -37,13 +54,75 @@ def test_render_files_template(self, client, sign_in):
         "event doesnt exist",
         "user not a participant in the event",
     ])
-    def test_invalid_event_id_in_file_url(self, client, sign_in, url_path, status_code):
+    def test_invalid_event_id_in_file_url(self, client, sign_in_with_creator, url_path, status_code):
         response = client.get(url_path)
         assert response.status_code == status_code
 
-    def test_unauthorized_user_redirected_to_main_page(self, client):
+    def test_unauthorized_user_redirected_to_login_page(self, client):
         response = client.get(f'{FILE_CREATION_URL}/1')
         assert response.status_code == 302
         response = client.get(response.url)
         assert response.status_code == 200
-        assertTemplateUsed(response, 'main/index.html')
+        assertTemplateUsed(response, 'login/register_login.html')
```

Comment:
Can you set the HTML path as a constant.
This one: 'login/register_login.html'

Reply:
Done